client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,53 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const bannerProps = sample( [
+	{
+		title: translate( 'Are you a photographer? An artist?' ),
+		description: translate(
+			'Explore {{b}}PHOTO BLOG{{/b}}, an elegant theme designed to showcase your visual masterpieces.',
+			{
+				components: {
+					b: <strong />,
+				},
+			}
+		),
+		action: () => {
+			alert( 'Photo Blog banner button clicked!' );
+		},
```

Comment:
We don't want to merge this with the `alert`, do we? 😄

Reply:
Haha No way! It's just there until I am instructed what kind of action are we going to do for the moment, if we're going to track the clicks, then we may use the `action`, or if we just want to go to a URL, then we just replace it with a plain `href`. I'll refactor this when I get the clear direction we're going with this.